js/src/data/actions.js

```diff
@@ -610,6 +610,21 @@ export function* requestPhoneVerificationCode( country, phoneNumber, method ) {
 			verificationId: response.verification_id,
 		};
 	} catch ( error ) {
+		// TODO: Remove the handling of 'backendError' and add the case for 'badRequest'.
+		//
+		// Currently, 'badRequest' won't be presented but maybe someday it can be handled.
+		// Ref:
+		// - https://github.com/woocommerce/google-listings-and-ads/issues/1101
+		// - https://github.com/woocommerce/google-listings-and-ads/issues/1998
```

Comment:
I'm unsure whether this should be marked as a "TODO" or considered pending since we lack knowledge about potential changes in the API response down the line. One option could be to omit the "TODO" label and simply leave the comments as they are.

Reply:
Removing only TODO might make the semantics deviate somewhat from what this error handling was supposed to do. I rephrased it in https://github.com/woocommerce/google-listings-and-ads/pull/1999/commits/cbda28317dd7547b4d6fc7fed62a6b4aa263045c to only describe the "why" but didn't specify "what" to do.